highlight.io/components/Home/FeatureCarousel/FeatureCarousel.tsx

```diff
@@ -31,6 +30,8 @@ import monitoringscreenshot from '../../../public/images/monitoringscreenshot.pn
 import openSource from '../../../public/images/open-source.png'
 import sessionReplay from '../../../public/images/session-replay.png'
 import sessionscreenshot from '../../../public/images/sessionscreenshot.png'
+import traces from '../../../public/images/traces.svg'
+import tracesscreenshot from '../../../public/images/tracesmobile.svg'
```

Comment:
Not a blocker - I noticed the other images are pngs, so curious why we switched to use an svg here?

Reply:
There was some discussion on the slack to move the images to svgs due to some blurriness, so I'm planning to switch all of them after this launch week crunch